reflex/app.py

```diff
@@ -329,6 +330,12 @@ def add_page(
         # Wrap the component in a fragment.
         component = Fragment.create(component)
 
+        # If component is not a Component object, try to unwrap it into a Fragment
+        if not isinstance(component, Component):
```

Comment:
if the line above wraps `component` in a fragment, then this condition will never be true.

Reply:
So I can delete it?

Reply:
@masenf I can delete it so

Reply:
I give it a try